spec/classes/config_spec.rb

```diff
@@ -273,7 +277,7 @@
         it 'appends setting to logback.xml' do
           is_expected.to contain_file('/etc/marathon/logback.xml')
             .with_content(
-              /\<logger name="mesosphere.chaos.http" level="ERROR"\/\>/
+              %r{\<logger name="mesosphere.chaos.http" level="ERROR"/\>}
```

Comment:
Rubocop will advise using `%r{}` for any regexes with `/`s in them so you don't need to escape them. I guess `/regex/` is kind of the default otherwise, though?